frontend/src/App.css

```
@@ -40,3 +40,324 @@
 .read-the-docs {
   color: #888;
 }
+
+/* ── Design tokens ──────────────────────────────────────────────────────────*/
```

Stylelint (17.3.0)

[error] Expected whitespace before "*/" (comment-whitespace-inside)

(comment-whitespace-inside)